src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Extentions/HttpClientExtension.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Altinn.App.PlatformServices.Extentions
+{
+    public static class HttpClientExtension
```

Comment:
Class and all methods are missing comments.

Reply:
Done